reference/docs-conceptual/getting-started/fundamental/Getting-Detailed-Help-Information.md

```diff
@@ -1,170 +1,203 @@
+To get a list of all the cmdlet Help articles in your session, type:
 
-```
-get-help -category cmdlet
+```powershell
+Get-Help -Category Cmdlet
 ```
 
-To display one page of each Help topic at a time, use the **help** function or its alias **man**. For example, to display Help for the Get-ChildItem cmdlet, type
+To display one page of each Help article at a time, use the `help` function or its alias `man`.
+For example, to display Help for the `Get-ChildItem` cmdlet, type
 
-```
-man get-childitem
+```powershell
+man Get-ChildItem
 ```
 
 or
 
-```
-help get-childitem
+```powershell
+help Get-ChildItem
 ```
 
-To display detailed information about a cmdlet, function, or script, including descriptions of its parameters and examples of its use, use the *Detailed* parameter of the Get-Help cmdlet. For example, to get detailed information about the Get-ChildItem cmdlet, type:
+To display detailed information, use the **Detailed** parameter of the `Get-Help` cmdlet. For
+example, to get detailed information about the `Get-ChildItem` cmdlet, type:
 
-```
-get-help get-childitem -detailed
+```powershell
+Get-Help Get-ChildItem -Detailed
 ```
 
-To display all content in the Help topic, use the *Full* parameter of the Get-Help cmdlet. For example, to display all content in the Help topic for the Get-ChildItem cmdlet, type:
+To display all content in the Help article, use the **Full** parameter of the `Get-Help` cmdlet. For
+example, to display all content in the Help article for the Get-ChildItem cmdlet, type:
 
-```
-get-help get-childitem -full
+```powershell
+Get-Help Get-ChildItem -full
 ```
 
-To get detailed Help about the parameters of a cmdlet, use the *Parameter* parameter of the Get-Help cmdlet. For example, to get detailed Help for all of the parameters of the Get-ChildItem cmdlet, type:
+To get detailed Help about the parameters of a cmdlet, use the **Parameter** parameter of the `Get-Help` cmdlet. For example, to get detailed Help for all of the parameters of the Get-ChildItem cmdlet, type:
 
-```
-get-help get-childitem -parameter *
+```powershell
+Get-Help Get-ChildItem -Parameter *
 ```
 
-To display only the examples in a Help topic, use the *Example* parameter of the Get-Help. For example, to display only the examples in the Help topic for the Get-ChildItem cmdlet, type:
+To display only the examples in a Help article, use the **Example** parameter of the `Get-Help`. For example, to display only the examples in the Help article for the Get-ChildItem cmdlet, type:
 
-```
-get-help get-childitem -examples
+```powershell
+Get-Help Get-ChildItem -examples
 ```
 
-For information about how to write Help topics for the cmdlets that you write, see [How to Write Cmdlet Help](https://go.microsoft.com/fwlink/?LinkID=123415) in the MSDN library.
+For information about how to write Help articles for the cmdlets that you write, see
+[How to Write Cmdlet Help](https://go.microsoft.com/fwlink/?LinkID=123415).
```

Comment:
Should the target of this link be migrated to docs?

Reply:
Had to find the new URL. 
Done